.circleci/config.yml

```diff
@@ -121,15 +134,62 @@ workflows:
   build_and_deploy:
     jobs:
       - build:
+          filters:
+            branches:
+              ignore:
+                - master
+                - /^support.*/
+                - /^feature\/.*-i18n/
+                - /^release.*/
+      - unit_test:
+          requires:
+            - build
+      - deploy_canary:
+          filters:
+            branches:
+              only: develop
+          requires:
+            - unit_test
+      - deploy_branch:
+          filters:
+            branches:
+              ignore:
+                - master
+                - develop
+                - /^support.*/
+                - /^feature\/.*-i18n/
+                - /^release.*/
+          requires:
+            - unit_test
+  build_and_deploy_i18n:
+    jobs:
+      - build_i18n:
           filters:
             tags:
               only: /^v.*/
+            branches:
+              only:
+                - master
+                - /^support.*/
+                - /^feature\/.*-i18n/
+                - /^release.*/
       - unit_test:
+          name: unit_test_i18n
```

Comment:
Is there some way we could simplify the filtering? I've always found it mentally annoying to piece together exactly what is happening, and we've definitely increased that complexity here. I think moving the unit test job into the build and build_i18n step could help a little (and I guess we would rename those steps slightly)

Reply:
actually eh it's probably fine I think the green diff on github makes it seem more confusing than it is

Reply:
it looks fine when I just look at the raw file